modules/RTC/TPCUtils.js

```diff
@@ -346,21 +490,25 @@ export class TPCUtils {
     }
 
     /**
-     * Returns the calculates max bitrates that need to be configured on the simulcast encodings based on the video
+     * Returns the calculates max bitrates that need to be configured on the stream encodings based on the video
      * type and other considerations associated with screenshare.
      *
      * @param {JitsiLocalTrack} localVideoTrack The local video track.
+     * @param {CodecMimeType} codec - The codec currently in use.
+     * @param {number} newHeight The resolution requested for the video track.
      * @returns {Array<number>}
      */
-    calculateEncodingsBitrates(localVideoTrack) {
+    calculateEncodingsBitrates(localVideoTrack, codec, newHeight) {
```

Comment:
The previous function is ` calculateEncodingsActiveState(localVideoTrack, newHeight, codec) {` can we please settle on an order for coded and height and use it consistently?